docs/layouts/shortcodes/generated/prometheus_push_gateway_reporter_configuration.html

```diff
@@ -12,7 +12,7 @@
             <td><h5>deleteOnShutdown</h5></td>
             <td style="word-wrap: break-word;">true</td>
             <td>Boolean</td>
-            <td>Specifies whether to delete metrics from the PushGateway on shutdown.</td>
+            <td>Specifies whether to delete metrics from the PushGateway on shutdown. but sometime metrics will not be deleted, see <a href="https://issues.apache.org/jira/browse/FLINK-13787">PrometheusPushGatewayReporter does not cleanup TM metrics when run on kubernetes</a></td>
```

Comment:
I'd like to emphasize in the docs that this config only "tries" to delete the metrics, but it is not guaranteed. For example

> Specifies whether to delete metrics from the PushGateway on shutdown. Flink will try its best to delete the metrics but this is not guaranteed. See <a href="https://issues.apache.org/jira/browse/FLINK-13787">here</a> for more details.

Reply:
thx, that's a good suggestion